### PR TITLE
Docs: Started Korean translation

### PR DESCRIPTION
**Pull request by hareha**
Sorry for confusing

rebased all

### CODE REVIEW

**Comment by Mugen87** [comment]
Korean is not yet selectable as a language. You have to add an entry here:

https://github.com/mrdoob/three.js/blob/d92c3085aa6e6c373d339a7fdd9bccf8b9e4c5f5/docs/index.html#L32-L36

And update the regex in:

https://github.com/mrdoob/three.js/blob/d92c3085aa6e6c373d339a7fdd9bccf8b9e4c5f5/docs/index.html#L75

**Comment by hareha** [comment]
modified select option and regex

File: docs/index.html [hunk 1]
```diff
@@ -33,6 +33,7 @@ <h1><a href="http://threejs.org">three.js</a></h1>
 						<option value="en">en</option>
 						<option value="ar">ar</option>
 						<option value="zh">中文</option>
+						<option value="kr">한국어</option>
```
Comment (Mugen87) [inline docs/index.html:36]: Please put `kr` between `ar` and `zh`.

Comment (hareha) [reply docs/index.html:35]: updated

**Comment by Mugen87** [comment]
Please change `kr` -> to `ko`, see https://github.com/mrdoob/three.js/pull/20877#discussion_r541711926.

**Comment by hareha** [comment]
changed kr to ko

File: docs/list.json [hunk 2]
```diff
@@ -998,6 +998,41 @@
...
 		}
 
+	},
+
+	"kr": {
+	
+		"매뉴얼": {
+	
+			"시작하기": {
+				"장면 만들기": "manual/kr/introduction/Creating-a-scene",
```
Comment (Mugen87) [inline docs/list.json:1008]: You are still using `kr` in those links.

Comment (hareha) [reply docs/list.json]: sorry.
modified now

**Comment by Mugen87** [comment]
`docs/manual/ko/introduction/Creating_a_scene.html ` is not named correctly. It should be:
```
docs/manual/ko/introduction/Creating-a-scene.html 
```
Currently, there is a 404 error because of the wrong naming:

https://raw.githack.com/hareha/three.js/korean/docs/index.html#manual/ko/introduction/Creating-a-scene

**Review by Mugen87 — APPROVED** [review]

**Comment by mrdoob** [comment]
Thanks!

**Pull request merged**